modules/tracking/include/opencv2/tracking/tracker.hpp

```diff
@@ -1366,9 +1370,9 @@ class ROISelector {
 };
 
 Rect2d CV_EXPORTS_W selectROI(Mat img, bool fromCenter = true);
-Rect2d CV_EXPORTS_W selectROI(const cv::String& windowName, Mat img, bool showCrossair = true, bool fromCenter = true);
-void CV_EXPORTS_W selectROI(const cv::String& windowName, Mat img, std::vector<Rect2d> & boundingBox, bool fromCenter = true);
-
+Rect2d CV_EXPORTS_W selectROI(const cv::String &windowName, Mat img, bool showCrossair = true, bool fromCenter = true);
+void CV_EXPORTS_W selectROI(const cv::String &windowName, Mat img, std::vector<Rect2d> &boundingBox, bool fromCenter = true);
+void CV_EXPORTS_W selectROI(const cv::String &windowName, cv::VideoCapture &cap, std::vector<Rect2d> &boundingBox, bool fromCenter = true);
```

Comment:
does it really need this function ? 

* imho, it's not practical to draw a boundingrect while the video moves, there's good reason to do this on a still image instead.

* the python wrappers complain about passing a reference to VideoCapture, so it probably should be CV_EXPORTS (w/o _W) , and so not wrapped to python